src/TMLeague/wwwroot/data/leagues/sl/seasons/s8/divisions/q1.json

```
@@ -19,6 +19,53 @@
     290189,
     290338
   ],
-  "penalties": [],
+  "penalties": [
+      {
+      "player": "PawcioF",
+      "game": 290240,
+      "points": 5,
+      "details": "talking in the open chat"
+    }
+      {
+      "player": "Ser Joseph",
+      "game": 290240,
+      "points": 7,
+      "details": "talking in the open chat, no delaying"
+    }
+      {
+      "player": "gauntlet",
+      "game": 290240,
+      "points": 15,
+      "details": "talking in the open chat, KM"
+    }
+      {
+      "player": "gauntlet",
+      "game": 290338,
+      "points": 5,
+      "details": "minor KM"
+    }
+      {
+      "player": "Shade89",
+      "game": 290240,
+      "points": 5,
+      "details": "minor KM"
+    }
+      {
+      "player": "Ser Joseph",
+      "game": 290193,
+      "points": 2,
+      "details": "no delaying"
+    }
+      "player": "Ser Joseph",
+      "game": 290259,
+      "points": 1,
+      "details": "no delaying"
+    }
```

Here even bracket is missing.
```suggestion
    },{
```